### ISSUE TITLE
Blue text after switching back from a Vim window

### ISSUE THREAD

**Issue by dimkr** [dimkr/tty8#2]
To reproduce:
- vim
- CTRL + J
- CTRL + K
- CTRL + K

The new shell's prompt is blue!

**Comment by barnumbirr** [comment]
Isn't happening for me with tty8 from latest@master.

**Comment by dimkr** [comment]
It happens if Vim draws the tilde characters in color - happens on my system: Trisquel 7, under xfce4-terminal.

**Comment by barnumbirr** [comment]
True. The tilde does get colored blue, I can't see any blue color in the other buffers though.
(on Debian 8.2 using xterm-256color).

**Comment by min4builder** [comment]
That's the terminal color leftover when you change ttys.
To fix it, just make tty8 echo `<ESC>[0m` after switching ttys (that's the ANSI sequence for "clear attributes")

**Comment by dimkr** [comment]
It's a bit more complicated than that: try to run vim in one terminal, then htop in another. Both work fine (with `write(1, "\033[0m", 4)`, right after `CLEAR_TTY`), until you switch from vi to htop: arrow keys no longer work and there's a block cursor at the top. It seems htop does not re-apply the missing attributes.

**Comment by min4builder** [comment]
I have no idea why this breaks htop, as that sequence clears only color & appearance attributes.
Anyways, it might be impossible to correctly implement this without emulating the terminal.

**Comment by dimkr** [comment]
Somehow, dtach is able to handle this case (as far as I know). I took a very close look at its code and used strace to see the full timeline of events - I see nothing self-evident that's missing in tty8 and explains this bug.
